jsconcat.php

```diff
@@ -31,8 +31,6 @@ function __construct( $scripts ) {
 			}
 			unset( $this->$key );
 		}
-
-		parent::__construct();
```

Comment:
That's exactly what I tried to describe in my comment :)

The question whether the whole implementation is correct is still open, imho. But again, I don't feel like it should be a subject of this bug fixing PR, but am open to discussion.